pages/common/mdp.md

```diff
@@ -0,0 +1,19 @@
+# mdp
+
+> A command-line based tool to make presentations from markdown files.
+
+- Launch a presentation from a markdown file in the terminal:
+
+`mdp {{presentation.md}}`
+
+- Disable fading transitions:
+
+`mdp --nofade {{presentation.md}}`
+
+- Invert font colors tu use within a light background terminal:
```

Comment:
typo: "tu". Also, may I suggest changing "within a light background terminal" --> "in terminals with light background"?